code/obj/blob.dm

```diff
@@ -968,7 +985,7 @@ var/image/blob_icon_cache
 /datum/projectile/slime
 	name = "slime"
 	icon = 'icons/obj/projectiles.dmi'
-	//state_overlay = "new_slime"
+	//state_overlay = "slime"
```

Comment:
commented out?

Reply:
i did a batch-rename of everything that was named "new_thing" for consistency

Reply:
ok its commented out in main so i stand by this change